classes/04class/iac/artifacts/example2/vpc.tf

```diff
@@ -0,0 +1,125 @@
+resource "aws_vpc" "devops_academy" {
+  cidr_block = "192.168.0.0/16"
+  tags = {
+    Name = "DevopsAcademy"
+  }
+}
+resource "aws_subnet" "subnet_private_a" {
+  vpc_id            = "${aws_vpc.devops_academy.id}"
+  cidr_block        = "192.168.10.0/24"
+  availability_zone = "ap-southeast-2a"
```

Comment:
I like the idea of being simple for an introduction to tf nd also not sure if will have time. But terraform provides you data resources to get the available azs on a given region.. will leave to you to consider or not 

https://www.terraform.io/docs/providers/aws/d/availability_zones.html

Reply:
it is a simple example, not production-ready. It is supposed to be more readable for beginners as a first contact with the tool.

Not sure if adding more functions will help to achieve this goal.

Reply:
done